test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -60,6 +64,7 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, new Integer(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+        assertNotUnique(null, "a", "b", null,"b");
```

Comment:
The expectation was to write a test case method. Note the naming convention for test methods (the first part is feature/method under test), second is the testing scenario and the third is the expected behavior. 
E.g. isAnyNull_emptyList_falseReturned
Part 2 and 3 of the test name can be removed to indicate an increased test scope